plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/connection/TcpConnection.java

```diff
@@ -67,6 +67,7 @@ protected DockerResponse request(String method, String path, String query, List<
         for (Pair<String, ?> header : headers) {
             connection.setRequestProperty(header.first, String.valueOf(header.second));
         }
+        connection.setRequestProperty("Host", url.toString());
```

Comment:
@garagatyi you are absolutely right. We should set it to host+port only. Although I've test it and it worked, path+query should not be there. I'll fix that.